app/models/comment.rb

```diff
@@ -1,5 +1,10 @@
 # A comment on belongs to a user and a post.
 class Comment < ApplicationRecord
-  belongs_to :user
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
   belongs_to :post
+
+  # A method that updates the comments counter for a post.
+  def update_counter_cache
+    post.update(commentsCounter: post.comments.count)
+  end
```

Comment:
- [ ] I think it would be nice to use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the `#update_comment_counter` method every time a new comment has been saved to `database`, then make the `update_comment_counter` method private 🙃 🙃 
The code can be similar to 👇 

```ruby
  after_save :update_comment_counter

  private

  def update_comment_counter
    author.increment!(:comments_counter)
  end
```